src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/GenericComponent.tsx

```diff
@@ -130,11 +130,9 @@ export function GenericComponent(props: IGenericComponentProps) {
     );
   }, [componentValidations]);
 
-  if (hidden) {
-    return null;
-  }
 
-  const handleDataUpdate = (value: any, key = 'simpleBinding') => {
+
+  const handleDataUpdate = React.useCallback((value: any, key = 'simpleBinding') => {
```

Comment:
Love the `useCallback`, but we still might want to include the deps array, no?

Reply:
Yes, it kind of the point, but I just wrapped the existing code, so it is hard to see. https://github.com/Altinn/altinn-studio/pull/7636/files/2ff7fff5e5e5fa93e9fce4b24eb04c4148252ee7#diff-da16fb0e92aed91d23afdb24b4d55b3d42b7b988f628b07494584e041d2bff38R178